### PR TITLE
Integration test to catch bad html

### PR DESCRIPTION
**Pull request by motdotla**

### CODE REVIEW

**Comment by elbuo8** [comment]
:+1:

**Comment by motdotla** [comment]
@devinus

File: lib/plugins/prefixer.js [hunk 1]
```diff
@@ -12,7 +12,12 @@ function PrefixPlugin(cheerioOptions, postcssOptions) {
 
 PrefixPlugin.prototype.run = function(html, callback) {
   try {
-    return callback(null, htmlAutoprefixer.process(html, this.cheerioOptions, this.postcssOptions));
+    var prefixed = htmlAutoprefixer.process(html, this.cheerioOptions, this.postcssOptions);
+    if (prefixed === '') {
+      return callback(new Error('Invalid html'));
```
Comment (devinus) [inline lib/plugins/prefixer.js]: Very small nitpick, but can you capitalize `HTML` :smile: Then you get a :+1:

Comment (elbuo8) [inline lib/plugins/prefixer.js]: No respect towards the nitpicks.

Shame on you @motdotla

Comment (motdotla) [inline lib/plugins/prefixer.js]: lol

**Pull request merged**